views/partials/header.ejs

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Cluck - Quiz 1</title>
+    <link rel="stylesheet" href="/css/bootstrap.min.css">
+    <link rel="stylesheet" href="/css/index.css">
+
+    <script src="/js/jquery-3.2.1.slim.min.js" charset="utf-8"></script>
+    <script src="/js/popper.min.js" charset="utf-8"></script>
+    <script src="/js/date.js" type="text/javascript"></script>
+
+    <script type="text/javascript">
+    window.onload = function()
+      {
+        tDate();
+      }
+    </script>
+
+    <script defer src="/js/bootstrap.min.js" charset="utf-8"></script>
+  </head>
+  <body>
+    <style>
+
+    </style>
+
+    <header class="container">
+    <nav class="navbar navbar-light">
```

Comment:
Your nav bar does not collapse when the window becomes small (As is required in part 5.4 `It must must collapse into a menu button when it's smaller than the sm breakpoint.`)